jsonrpc/filter_manager.go

```diff
@@ -339,6 +345,70 @@ func (f *FilterManager) Exists(id string) bool {
 	return ok
 }
 
+// GetFilterLogs returns an array of logs for the specified filter.
+func (f *FilterManager) GetFilterLogs(id string) ([]*Log, error) {
```

Comment:
I guess it will take much time to return data if the chain is long. Is there any limitation?

Reply:
Not that I know of, but will test this with the longest chains we have. I do not see any other option for writing this with our code archistructure

Reply:
Completely changed the implementation so this is no longer a problem